frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/utilization-card/utilization-card.tsx

```diff
@@ -42,6 +45,26 @@ const UtilizationCard: React.FC<DashboardItemProps> = ({
     };
   }, [watchPrometheus, stopWatchPrometheusQuery, duration]);
 
+  const storageUtilization = prometheusResults.getIn([
```

Comment:
```suggestion
  const capacityUtilization = prometheusResults.getIn([
```

Reply:
I find capacity more appt, can you change all occurrences.

Reply:
not an issue.